src/components/LazyLoadImage.jsx

```diff
@@ -58,26 +66,31 @@ class LazyLoadImage extends React.Component {
   }
 
   getWrappedLazyLoadImage(lazyLoadImage) {
-    const { effect, height, placeholderSrc,
+    const { effect, height, placeholderProps, placeholderSrc,
       width, wrapperClassName } = this.props;
     const { loaded } = this.state;
 
     const loadedClassName = loaded ?
       ' lazy-load-image-loaded' :
       '';
+    const props = {
+      ...placeholderProps,
+      style: {
+        backgroundImage: loaded ? '' : 'url( ' + placeholderSrc + ')',
```

Comment:
Maybe it will be better to put these styles somewhere in the css, so that it will be enough to pass `className` in the `placeholderProps` (and merge classes here) to override styles without `important` hacks

Reply:
Also it will simplify applying different styles for different resolutions using media queries

Reply:
Thanks for taking a look @BilyachenkoOY ! The issue is that we need to read `placeholderSrc` from the props, and that can't be done if we move the styles completely to a CSS file. It's true we could keep `backgroundImage` in the `style` while moving everything else to a CSS file, but I think that would be confusing.

In the case you need to make a lot of modifications to the placeholder style, probably it's a better idea passing a node as the `placeholder` prop instead of adding props to the default placeholder. How does that sound?

Reply:
I still believe that there should be as few as possible in props and it will be fine if there will be only `backgroundImage` + height&width since it should be as easy as possible to override styles using `wrapperClassName`.

As for `placeholder `, it will be placed inside this `<span>` and is not suitable for all style manipulations.

It is not a big issue to me to add few `important` directives to my CSS, so it is up to you